core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -0,0 +1,50 @@
+
+// Notice!
+//
+// If you update this file, you *must* be sure to re-run
+//
+//  core/tools/ldml-const-builder/build.sh clean build run
+//
+// To update keyboardprocessor_ldml.h, and commit the result.
+//
+// It is not updated automatically.
+
+// namespace com.keyman.core.ldml {
+    /**
+     * Constants shared between the data access layer and the compiler
+     */
+    export const constants = {
+        /**
+         * The version of the LDML processor
+         */
+        version: '1.0',
+
+        section_keys: 'keys',
```

Comment:
We should add a comment above these constants. Do we need any reference to kmxplus in this file?

Reply:
Adding a comment, and also better description up top